src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -720,22 +724,22 @@
     public void setJASSConfigurationName(String configurationName);
 
     /**
-     * Retrieves the login configuration file for Kerberos authentication.
+     * Returns the login configuration file for Kerberos authentication.
      * 
      * @return login configuration file name
      */
     public String getJASSConfigurationName();
 
     /**
-     * Enables Fips Mode on the connection For FIPS enabled JVM this property should be true.
+     * Sets Fips Mode on the connection For FIPS enabled JVM this property should be true.
```

Comment:
"Sets Fips Mode" does not make much sense. I would change it to "Sets the value to enable/disable FIPS Mode..."